tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,2 +1,32 @@
-frost-sidebar testbed
-{{frost-sidebar}}
+     {{#frost-tabs class=(if iframe 'iframe') on-change=(action 'tabSelected') selection=selectedTab}}
+  {{#frost-tab alias='Readme' id='readme'}}
```

Comment:
Please remove frost-tabs from the demo and package. We aren't using tabs anymore. Take a look at any of the core demos like select or button for example demos. There will be styles in the app.scss you can use for the layout for the demo